reviews/models/mixins.py

```diff
@@ -267,8 +267,11 @@ def reviews_notification(self, context):
 @reviews_signals.reviews_email_submit.connect
 def reviews_submit_notification(self, context):
     template = context['template']
+    event_type = utils.find_subscription_type('global_reviews')
     for user_id in context['email_recipients']:
         user = OSFUser.load(user_id)
+        user_subscriptions = get_user_subscriptions(user, event_type)
```

Comment:
I think we should maybe use `compile_subscriptions` here, to allow for node subscription settings? And then check `user_id in subscriptions['none']` to determine whether they'll be getting future emails. @sloria, does that make sense?

Reply:
Hm, yeah this is a little weird. If the user is creating a preprint from an existing project this could be inaccurate.

Reply:
I'm not sure if `compile_subscriptions` is correct here. Are review emails configurable on nodes? If not, then I think this is correct.

Reply:
They will be configurable on nodes [shortly](https://openscience.atlassian.net/browse/MOD-150).

Reply:
They are configurable on nodes (or will be soon). Ideally they would be configured on preprints instead of nodes though. Once the node/preprint divorce happens this setting won't make any sense on projects.